lang/smaily-for-wp-et.po

```diff
@@ -1,9 +1,9 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: Smaily for WP\n"
-"POT-Creation-Date: 2021-02-03 12:58+0200\n"
-"PO-Revision-Date: 2021-02-03 12:58+0200\n"
-"Last-Translator: Smaily <info@smaily.com>\n"
+"POT-Creation-Date: 2021-02-18 11:11+0200\n"
+"PO-Revision-Date: 2021-02-18 11:11+0200\n"
+"Last-Translator: Tom <tom@example.com>\n"
```

Comment:
Make sure this is `Smaily <info@smaily.com>`.

Reply:
Yep. I didn't want to change this. c8a47da